modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,121 @@
+
+* This enhancement updates the selectors for all components to use common Kubernetes labels. (link:https://issues.redhat.com/browse/LOG-5906[LOG-5906])
+
+* With this update, the collector configuration now deploys as a ConfigMap instead of a secret. This change allows users to view and edit the configuration when the collector is not managed. (link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
+* With this release, the log level for the Vector collector can be configured using an annotation on the ClusterLogForwarder. The log level options include trace, debug, info, warn, error, or off. (link:https://issues.redhat.com/browse/LOG-5372[LOG-5372])
+* This enhancement adds validation to reject configurations where CloudWatch outputs use multiple AWS roles. Previously, logs were incorrectly routed to the first AWS role defined. (link:https://issues.redhat.com/browse/LOG-5640[LOG-5640])
+* This enhancement removes the *Log Bytes Collected* and *Log Bytes Sent* graphs from the metrics dashboard. (link:https://issues.redhat.com/browse/LOG-5964[LOG-5964])
+* With this release, the must-gather functionality is updated to only capture information for inspecting Logging 6.0 components, including Vector deployments from `ClusterLogForwarder.observability.openshift.io` resources and the Red Hat managed `LokiStack` provided by the Loki Operator.
+
+=== Log Storage
+
+* With this update, the Azure storage secret validation now provides early warnings for specific error conditions. (link:https://issues.redhat.com/browse/LOG-4571[LOG-4571])
+* Before this update, the Loki Operator configured Loki to use path-based style access for the Amazon Simple Storage Service (S3), which has been deprecated. With this update, the Loki Operator defaults to virtual-host style without users needing to change their configuration. (link:https://issues.redhat.com/browse/LOG-5051[LOG-5051])
+* Before this update, the Loki Operator did not validate the Amazon Simple Storage Service (S3) endpoint used in the storage secret. With this update, the validation process ensures the S3 endpoint is a valid S3 URL, and the `LokiStack` status updates to indicate any invalid URLs. (link:https://issues.redhat.com/browse/LOG-5052[LOG-5052])
+
+* With this release, the responsibility for deploying the logging view plugin shifts from the ClusterLoggingOperator to the ClusterObservabilityOperator. For new log storage installations that need visualization, the ClusterObservabilityOperator and the associated UIPlugin resource must be deployed. Refer to the official product documentation for more details. (link:https://issues.redhat.com/browse/LOG-5461[LOG-5461])
```

Comment:
I think this should maybe be included in the Collector release notes since that was the component that was responsible for visualization. It should also be one of the first topics to show on the list since it has a high impact

Reply:
I am not sure what's collector release notes but I will move this to the top. Thanks!